src/quo2/components/list_items/dropdown.cljs

```diff
@@ -0,0 +1,148 @@
+                                                        :padding-horizontal 8}
+                                 :padding-with-no-icon {:padding-vertical 5
+                                                        :padding-left     12
+                                                        :padding-right    8}}}
+            :small  {:icon-size 12
+                     :font      {:font-size :label}
+                     :height    24
+                     :padding   {:padding-with-icon    {:padding-vertical   3
+                                                        :padding-horizontal 6}
+                                 :padding-with-no-icon {:padding-vertical   3
+                                                        :padding-horizontal 6}}}})
+(defn color-by-10
+  [color]
+  (colors/alpha color 0.6))
+
+(defn dropdown-comp [{:keys [icon open? dd-height size disabled? dd-color border-color]}]
+  (let [dark?                                                         (colors/dark?)
+        {:keys [width height width-with-icon padding font icon-size]} (size sizes)
+        {:keys [padding-with-icon padding-with-no-icon]}              padding
+        font-size                                                     (:font-size font)
+        spacing                                                       (case size
+                                                                        :big 4
+                                                                        :medium 2
+                                                                        :small 2)]
+    [rn/touchable-opacity (cond-> {:on-press (fn []
+                                               (if (swap! open? not)
+                                                 (apply-anim dd-height 120)
+                                                 (apply-anim dd-height 0)))
+                                   :style    (merge
+                                              (if icon
+                                                padding-with-icon
+                                                padding-with-no-icon)
+                                              {:width            (if icon
+                                                                   width-with-icon
+                                                                   width)
+                                               :height           height
+                                               :border-radius    (case size
+                                                                   :big 12
+                                                                   :medium 10
+                                                                   :small 8)
+                                               :flex-direction   :row
+                                               :align-items      :center
+                                               :border-width     1
```

Comment:
I think the `border-width` affects the height and width of a component as it "eats" into the height and width. 
Is it better to wrap in a separate view component with the border set there as that it will add the border on and it easier to calculate.
@Parveshdhull I recall you discussing this before. WDYT?

Reply:
component has inner border in figma, so its ok in this case.
![image](https://user-images.githubusercontent.com/17097240/191245871-d5f7997b-3230-411a-a6e2-364144d5df44.png)

Reply:
ah, I didn't know about this. Thanks Parvesh 👍